lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +281,64 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._mystem = Mystem()
+        mapping_path = Path(__file__).parent / 'data' / 'mystem_tags_mapping.json'
+        self._converter = MystemTagConverter(mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentences = []
+        result = (i for i in self._mystem.analyze(text))
+        for sentence_position, sentence in enumerate(split_by_sentence(text)):
+            conllu_tokens = []
+            original_sentence = sentence
+
+            tokens = []
+            for token in result:
+                if token['text'] not in sentence:
```

Comment:
do you suggest for each sentence to go through all text and skip unrelated words? and unrelated means not a duplicate of this word? how do you prove that this word is not a duplication and is exactly from this sentence?